uwrt_mars_rover_drivetrain/uwrt_mars_rover_drivetrain_description/config/physical_parameters.yaml

```diff
@@ -0,0 +1,14 @@
+chassis:
+  length: 1.5
+  width: 1.0
+  height: 0.15
+
+wheels:
+  # Should match values defined in uwrt_mars_rover_drivetrain_hw/config/drivetrain_controllers.yaml
+  diameter: 0.3
+  width: 0.2
+  mass: 0.5
+
+control:
+  mechanical_reduction: 0.43
+  max_speed: 10
```

Comment:
TODO: all these variables need to be updated to values provided by mech

Reply:
![image](https://user-images.githubusercontent.com/16629657/149639008-33b2eebc-4500-4653-a178-9e9a2225d4cd.png)

Reply:
updated